src/library_webgpu.js

```diff
@@ -40,19 +40,9 @@
       return s;
     },
 
-    makeI32I32ToU53: function(lowName, highName) {
-      var ret = '(';
-      if (ASSERTIONS) {
-        ret += `assert((${highName} >>> 0) < 0x200000), `;
-      }
-      ret += `(${highName} >>> 0) * 0x100000000 + (${lowName} >>> 0))\n`;
```

Comment:
aside: This was actually equivalent to `convertU32PairToI53` rather than `convertI32PairToI53`. But I think the difference is only when `hi` is outside the [0, INT32_MAX] range, so this should be fine.